app/authentication/session_storage.py

```diff
@@ -30,9 +31,9 @@ def store_user_id(user_id):
             # session has a add function but it is wrapped in a session_scope which confuses pylint
             db_session.add(eq_session)
 
-    def clear(self):
+    def delete_session_from_db(self):
         """
-        Removes a user id from the session
+        deletes user session from database
```

Comment:
The main code this function is calling is db_session.delete(eq_session). So I have just aligned the function name, clear was a rubbish, it confused me as I thought it was clearing the whole contents of the session, wrong